tabbycat/templates/components/item-action.html

```diff
@@ -3,18 +3,18 @@
   <a href="{{ url }}" {% if blank %}target="_blank"{% endif %} {% if id %}id="{{ id }}"{% endif %}
    class="list-group-item list-group-item-action
    {% if to_complete %}
-    bg-primary text-white
+    bg-{% if type %}{{ type }}{% else %}primary{% endif %} text-white
```

Comment:
just pinging @philipbelesky in case you'd object to this (and change below in the same file)

Reply:
Hmm, I think the intent of the `to_complete` flag was to standardise on the primary colour as the indicator for 'key todo action'. Not fundamentally opposed to having an override to that if its used sparingly

Reply:
Of course. In this particular case it's security implications that I'd argue warrant the override; I can't think of any others right now, but in my head the threshold is something like "OH MY GOD THIS IS WEIRD AND DANGEROUS YOU DEFINITELY WANT TO CHECK THIS FIRST". (Not just, "You should do this!")

If you want to restructure to _e.g._ make it a different argument (to prevent mishaps or misuse) I'd be fine with that too.